src/web/src/pages/Main.vue

```diff
@@ -310,9 +310,10 @@ export default {
             exclusive_date_end.setDate(course.date_end.getDate() + 1);
             semester = session.semester;
             calendarBuilder.addEvent(
-              `Class: ${course.title}`,
+              `Class: ${course.full_title || course.title}`,
```

Comment:
Let's remove the Class prefix, given most of us are full time students, there's no need to specify context in the name of the event.